src/app/components/close-popup/close-popup.html

```diff
@@ -1,4 +1,4 @@
-<ion-button fill="clear" (tap)="onClose()" [color]="color">
+<ion-button fill="clear" (click)="onClose()" [color]="color">
```

Comment:
Awesome work, man :3

Just a doubt here, why we are changing from `tap` to `click`? The click makes the action fired immediately but could bring some problems on mobile, isn't better to switch to `press` instead?

Reply:
First review :noah:

This tap stuff is an old fix when Ionic 2 did not handle it properly, so it was handled by hammer.js. But it has now been fixed and this close button is not working atm.